### DISCUSSION TITLE
Add other tiptap extensions

### DISCUSSION THREAD

**Discussion by formfcw** [formfcw/directus-extension-flexible-editor#6] [Feature Request]
Not all extensions have been implemented yet: https://tiptap.dev/extensions

If you are planning to integrate a specific extension, please leave a comment here so we can discuss whether the extension would be useful or not.

**Comment by jdjoshuadavison** [comment]
The two extensions that we'd most like to see added to Flexible Editor is the Collaborator and Collaborator Cursor extensions.

**Comment by jdjoshuadavison** [comment]
We'd love to have the Character Count extension added.

**Comment by jdjoshuadavison** [comment]
Currently we are using relational nodes to insert images, but it would be nice to have the an actual image block that just works with the file browser directly.

**Comment by karthikktamilmani-freshworks** [comment]
I would like to integrate with the textAlign extension. I believe it would be very useful in improving the formatting and alignment options for our content. - https://tiptap.dev/api/extensions/text-align

**Comment by formfcw** [comment]
Thank you for your interest!

Feel free to contribute. Here is a comment about adding extensions: https://github.com/formfcw/directus-extension-flexible-editor/discussions/27#discussioncomment-7241970
